### PR TITLE
[NEAT-492] 🦟Core import bug

### PR DESCRIPTION
**Pull request by doctrino**
## Before
image

## After
image

### CODE REVIEW

File: cognite/neat/_rules/models/dms/_exporter.py [hunk 1]
```diff
@@ -395,6 +400,43 @@ def _gather_properties(
 
         return container_properties_by_id, view_properties_by_id
 
+    @staticmethod
+    def _gather_properties_with_parents(
```
[the remark this answers is not in the document]
Comment (nikokaoja) [reply cognite/neat/_rules/models/dms/_exporter.py:404]:
```suggestion
    def _gather_properties_with_ancestors(
```

[the remark this answers is not in the document]
Comment (nikokaoja) [reply cognite/neat/_rules/models/dms/_exporter.py:404]: Leverage `cognite.neat._utils.rdf_.get_inheritance_path`

**Review by nikokaoja — COMMENTED** [review]
See my comments on naming things

**Review by nikokaoja — CHANGES_REQUESTED** [review]
See my comments on naming things and using util function for inheritance

File: cognite/neat/_rules/models/dms/_exporter.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (doctrino) [reply cognite/neat/_rules/models/dms/_exporter.py:404]: The difference is that here you also set the child view as the view for all properties. Do you still think we should expose it in the DMSAnalysis or keep it as a private implementation detail?

**Pull request merged**